src/lexers/javascript-lexer.js

```diff
@@ -10,35 +10,32 @@ const WalkerBase = Object.assign({}, walk.base, {
   }
 })
 
+const defaultInjectors = [injectAcornEs7, injectAcornStage3]
+const defaultPlugins = { stage3: true, es7: true }
```

Comment:
I would get rid of this personally and document what could be a good default setup in the README.

Reply:
Sounds good to me.